redis/asyncio/sentinel.py

```diff
@@ -230,15 +231,24 @@ async def execute_command(self, *args, **kwargs):
         if "once" in kwargs.keys():
             kwargs.pop("once")
 
+        # Check if command suppose to return boolean response.
+        bool_resp = bool(kwargs.get("bool_resp", False))
+        if "bool_resp" in kwargs.keys():
+            kwargs.pop("bool_resp")
+
         if once:
-            await random.choice(self.sentinels).execute_command(*args, **kwargs)
-        else:
-            tasks = [
-                asyncio.Task(sentinel.execute_command(*args, **kwargs))
-                for sentinel in self.sentinels
-            ]
-            await asyncio.gather(*tasks)
-        return True
+            return await random.choice(self.sentinels).execute_command(*args, **kwargs)
+
+        tasks = [
+            asyncio.Task(sentinel.execute_command(*args, **kwargs))
+            for sentinel in self.sentinels
+        ]
+        responses = await asyncio.gather(*tasks)
+
+        if bool_resp:
```

Comment:
I think we need to change the behaviour here to be backward compatible. The default behaviour should be still "return True", if you want to return the actual responses you have to explicitly specify this.

So instead we would likely to have something like `return_response=True` flag and we will return actual response if it will be explicitly set.